history/history-impl/src/main/java/com/duckduckgo/history/impl/HistoryRepository.kt

```diff
@@ -90,6 +95,14 @@ class RealHistoryRepository(
         }
     }
 
+    override fun isHistoryUserEnabled(default: Boolean): Boolean {
```

Comment:
I decided to add it here as a way to have the repo combining the different data sources, but it also feels a bit odd to expose the default value, so I can change it. No strong opinions

Reply:
Yes, I need the value to change based on the RC flag (If the local value is null, then default to the RC flag value). The reason I added it there is because I'm considering handling that case as part of the business logic. Let me know what you think